controllers/reconcilers/kserve_authconfig_reconciler.go

```diff
@@ -56,6 +57,10 @@ func NewKserveAuthConfigReconciler(client client.Client, scheme *runtime.Scheme)
 }
 
 func (r *KserveAuthConfigReconciler) Reconcile(ctx context.Context, log logr.Logger, isvc *kservev1beta1.InferenceService) error {
+	if !utils.IsGroupRegisteredInScheme(r.client, authorinov1beta2.GroupVersion) {
+		log.V(1).Info("Skipping authorization reconciliation, authorization capability is not set up")
+		return nil
+	}
```

Comment:
wanna make sure i understand this part:
so if odh-model-controller is only used with modelserving (as no kserve) what would happen here?

Reply:
Are you concerned about the log being confusing? This reconciler is only active when KServe is enabled so I think we should be good here.

Reply:
Shouldn't we check `isAuthorinoEnabled` here as well or something similar?
AFAIK we check if the Service Mesh is enabled to allow KServe raw deployments

Reply:
we do not have an option to set authornio to Removed or Unmanaged in operator
as long as it is installed it will be enough.

Reply:
I agreed with @spolti because authorino can be installed for other reasons by users. so we should double check it. [VerifyIComponentIsEnabled](https://github.com/opendatahub-io/odh-model-controller/blob/5fc41fb3ce6730aa628bb9a588d34a3e442d0a86/controllers/utils/utils.go#L80) func that @spolti implemented can verify if kerve serverless installed or not that require Authorino operator or not. Why don't you add the func IsGroupRegisteredInScheme into this func and use VerifyIComponentIsEnabled func?

Reply:
I changed the logic, PTAL at new version